examples/cop-dem-glo-30/Copernicus_DSM_COG_10_N53_00_W115_00_DEM/Copernicus_DSM_COG_10_N53_00_W115_00_DEM.json

```
@@ -73,6 +78,12 @@
       "title": "Copernicus DEM User handbook",
       "description": "Also includes data usage information"
     },
+    {
+      "rel": "product handbook",
+      "href": "https://spacedata.copernicus.eu/documents/20123/122407/GEO1988-CopernicusDEM-SPE-002_ProductHandbook_I5.0+%281%29.pdf/706ee17d-2cce-f1fa-a73e-1686d28f09dd?t=1679657087883application/pdf",
+      "type": "Compernicus DEM Product Handbook (Nov 2022)",
```

```suggestion
      "type": "Copernicus DEM Product Handbook (Nov 2022)",
```

Perhaps also change "handbook" to "Handbook" on line 78 (a few lines above, for "title")